api/proto/metrics/metrics.proto

```diff
@@ -140,9 +182,14 @@ message appContainerMetric {
   string status = 2;           // Status string e.g. Uptime 3 hours
   uint32 PIDs = 3;             // Number of PIDs inside the container
   appCpuMetric cpu = 4;        // container cpu usage
+
+  // memory - OBSOLETE. Use memory_v2 instead
   memoryMetric memory = 5;     // container memory usage, will fill 'usedMem', 'availMem' initially
+
   networkMetric network = 6;   // container network usage, will fill 'txBytes', 'rxBytes' initially
   diskMetric disk = 7;         // container Block IO, will fill 'readBytes', 'writeBytes' initially
+
+  AppMemoryMetric memory_v2 = 8;
```

Comment:
Can we call this field appContainerMemory instead?

Reply:
Done.